libethdrivers/src/plat/imx6/uboot/imx-regs.h

```diff
@@ -33,6 +33,9 @@
 #define ROMCP_ARB_BASE_ADDR             0x00000000
 #define ROMCP_ARB_END_ADDR              0x000FFFFF
 
+#if defined (CONFIG_PLAT_IMX8MQ_EVK)
+    #define CONFIG_PLAT_IMX6DQ
```

Comment:
This is semantically broken, if you need things to work on i.MX8, then don't abuse `CONFIG_PLAT_IMX6DQ`, but extend the code to also support `CONFIG_PLAT_IMX8MQ`.

Reply:
@kent-mcleod  I don't feel I am the person to defend this? It works, I am all in favor of being able to build code that works, without this patch it doesn't.

Reply:
These changes were the minimum set to be able to get the driver running on the imx8mq.  They would need a round of cleanup before they could be merged.  

This driver used to support the imx8mq platform too (https://github.com/seL4/util_libs/blob/master/libethdrivers/CMakeLists.txt#L68).  I believe when the changes for adding solox were made the support for imx8mq was unintentionally removed.  These changes are a placeholder for what changes would be needed to add imx8mq support back in.

Reply:
Can you add a comment in the code that this is a quick hack that is to be replaced then? If there is a i.MX8 board for testing available now I think the time should be take to add this cleanly before it gets merged. Having a `CONFIG_PLAT_IMX8MQ` set up in CMake seems useful in the ling run.